base/js/admin.js

```diff
@@ -788,12 +796,18 @@ var sowbForms = window.sowbForms || {};
 						removeItem();
 					} else if ( confirm( soWidgets.sure ) ) {
 						$item.slideUp('fast', removeItem );
+						if($('.siteorigin-widget-field-repeater-add').hasClass('is-hidden')) {
+                            $('.siteorigin-widget-field-repeater-add').show();
+                            $('.siteorigin-widget-field-repeater-add').removeClass('is-hidden');
+                            $('.siteorigin-widget-field-copy').show();
+						}
 					}
 				});
 				itemTop.find('.siteorigin-widget-field-copy').click(function (e) {
 					e.preventDefault();
 					var $form = $(this).closest('.siteorigin-widget-form-main');
 					var $item = $(this).closest('.siteorigin-widget-field-repeater-item');
+                    var $maxItemsRepeater = $('.siteorigin-widget-field-repeater').attr('data-max-items');
```

Comment:
This could actually just be `$('.siteorigin-widget-field-repeater').data('max-items')`, which is slightly neater. Also, rather than casting `$maxItemsRepeater` to an integer later on, this could be done on this line.

We're also using `$` in front of a variable name to represent a jQuery object. So in this particular case, the variable should just be `maxItemsRepeater`.